sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -405,6 +440,19 @@ def test_series_agg(self):
       self._run_test(lambda s: s.agg(['mean']), s)
       self._run_test(lambda s: s.agg('mean'), s)
 
+  def test_append_sort(self):
+    df1 = pd.DataFrame({'int': [1, 2, 3], 'str': ['a', 'b', 'c']},
```

Comment:
yapf insists on really messing this up:
```suggestion
    df1 = pd.DataFrame({
        'int': [1, 2, 3], 'str': ['a', 'b', 'c']
    },
```

I'd rather keep it as-is but not sure how to make yapf happy.

Reply:
You can tag a line with `# yapf: disable`

Reply:
Thanks!